app/src/main/AndroidManifest.xml

```diff
@@ -19,23 +19,24 @@
 
 
     <application
-            android:allowBackup="true"
-            android:icon="@mipmap/ic_launcher"
-            android:label="@string/app_name"
-            android:logo="@drawable/golem_eye"
-            android:resizeableActivity="true"
-            android:supportsRtl="true"
-            android:theme="@style/AppTheme"
-            tools:targetApi="n">
+        android:name="de.eknoes.inofficialgolem.MainApplication"
```

Comment:
Ich bin nicht mehr so in der Android-Entwicklung drin, aber (warum) brauchen wir das, und die neue Klasse? :)

Reply:
Ich bin auch eigentlich Java Backend-Entwickler😅 und habe das einfach Mal nen bissl zum Spaß angefangen. Also bin ich auch für jeden Hinweis dankbar. 
Das brauchen wir aber um die Applikation zu ziehen. Ich schick dir nachher Mal den CP wofür ich das gemacht habe.